Applications/Core TLP.md

```diff
@@ -0,0 +1,70 @@
+## Introduction
+
+Currently the Node.js Foundation directly oversees the development of the core platform and associated working groups. The addition of Top Level Projects will predictably increase the workload on the TSC and so it makes sense to transfer these responsibilities to a Top Level Project concerned solely with the development of the platform.
+
+## History & Metrics
+
+Created by Ryan Dahl, now the largest programming ecosystem on the planet.
+
+## Scope
+
+The Core TLP will have sole responsibility and discretion over the node.js project in the following areas:
+
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Governance process and practices.
+* Contribution process and practices.
+* Maintaining the list of additional Collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between Collaborators and Working Groups.
```

Comment:
the collaboration wg is there to share knowledge about how to best collaborate and to build tools that help us do so, the TC or WG responsible for each repo is ultimately the arbiter of conflicts. this line delegates that responsibility from the TSC (where it current resides because of the TSC Charter passed by the foundation's board of directors) to the Core TLD for the "node.js" project.